### PR TITLE
Normalized Store

### PR DESCRIPTION
**Pull request by thudsonbu**
This pr adds store slices for topics, takeaways, action items, meetings, and participants  using a slice generator util I wrote. The generator allows for the stores to be normalized meaning that each is structured in a similar way as collections on the backend and there is thus no duplicated data. In Addition, since it would be inefficient to search the local store for items related to a parent item (like takeaways to a parent topic) the topic is given a `takeaways` array that includes the ids of related topics. Since items in the store are stored in the form `[_id]: { ...object }`, the lookup of id's in the store is far more efficient. The generator also creates commonly used selectors and actions so you can do things like:

```js
import meetingStore from "../store/features/meeting";

const meeting = useSelector((state) => meetingStore.selectors.get(meeting_id, state));
const topics = useSelector((state) => meetingStore.selectors.topics(meeting_id, state));

const onUpdate = () => {
  dispatch(meetingStore.actions.update({ ...updates });
};
```

I think there is room for improvement in terms of schema validation but it does allow us to reference the same data in multiple places without making multiple (often heavier) calls like were had previously been.

### CODE REVIEW

File: api/test/utils/client.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const axios = require("axios");
 
 const client = axios.create({
-  baseURL: "http://localhost:5000/api",
+  baseURL: "http://localhost:5555/api",
```
Comment (zbarnz) [inline api/test/utils/client.js:4]: Did we run into something on port 5000? Youre not on a mac are you????? 🤮

File: www/components/pages/Home/Inbox/Inbox.js [hunk 2]
```diff
@@ -32,8 +32,6 @@ export default function Inbox({ meetings, refresh }) {
     // and topics have a meeting_id to reference
     const res = await meetingAPI.create({ name: "Draft", date: new Date() });
 
-    console.log("create res", res);
```
Comment (zbarnz) [inline www/components/pages/Home/Inbox/Inbox.js:35]: The last of his kind

File: api/test/utils/client.js (hunk 1 above)
Comment (thudsonbu) [reply api/test/utils/client.js:4]: Yeah, they took over port 5000 on my new laptop. These are tough times.

**Comment by zbarnz** [comment]
bruh i wasnt done

**Pull request merged**